### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by Bernie1410**
adds missing german terms

like  "quest_34": "Erhalte %{amount} Herzen mit deinem Kumpel",

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: resources/webroot/static/data/de.json [hunk 1]
```diff
@@ -3219,8 +3284,10 @@
 		"quest_28": "Nehme %{amount} Schnappschuss/Schnappschüsse",
 		"quest_29": "Kämpfe in %{amount} Team Rocket Kampf/Kämpfe",
 		"quest_30": "Erlöse %{amount} Pokemon",
-        "quest_31": "Find %{amount} Team Rocket Invasion(en)",
-        "quest_32": "%{amount} mal erster Grunt des Tages",
+        	"quest_31": "Find %{amount} Team Rocket Invasion(en)",
+        	"quest_32": "%{amount} mal erster Grunt des Tages",
+		"quest_33": "Unbekannt",
+        	"quest_34": "Erhalte 5 Herzen mit deinem Buddy",
```
Comment (123FLO321) [inline resources/webroot/static/data/de.json]: shouldn't this be %{amount} instead of 5?

Comment (Bernie1410) [reply resources/webroot/static/data/de.json]: yes, now it's correct

**Comment by whitewillem** [comment]
Still contains tabs. Fix it and ill approve.

File: resources/webroot/static/data/de.json [hunk 2]
```diff
@@ -76,17 +80,78 @@
...
+        	"profile_update_password_success": "Passwort erfolgreich geändert.",
+        	"email_confirm_subject": "Willkommen %{name}!",
+        	"email_confirm_title_1": "Hallo %{name},",
+        	"email_confirm_title_2": "Willkommen %{name}!",
+        	"email_confirm_subtitle": "Um die Anmeldung zu beenden bitte bestätige deine Email Adresse durch klicken auf den Button unten.\nDer Link ist 7 Tage gültig.",
+        	"email_confirm_button": "Bestätige Email",
+        	"email_reset_subject": "Passwort von %{name} zurücksetzen!",
+        	"email_reset_title_1": "Hallo %{name},",
+        	"email_reset_title_2": "Passwort von %{name} zurücksetzen!",
+        	"email_reset_subtitle": "Du hast eine Passwort Löschung beantragt. Nutze den Button unten um dein Passwort zu löschen.\nFalls du keine Passwort Löschung beantragt hast, ignoriere diese Email\nDieser Link ist eine Stunde gültig.",
```
Comment (PapileinDE) [inline resources/webroot/static/data/de.json]: Ich denke es sollte eine Passwort Rücksetung heißen.

Comment (123FLO321) [reply resources/webroot/static/data/de.json]: fixed

File: resources/webroot/static/data/de.json [hunk 3]
```diff
@@ -3220,6 +3285,8 @@
         "quest_30": "Erlöse %{amount} Pokemon",
         "quest_31": "Find %{amount} Team Rocket Invasion(en)",
         "quest_32": "%{amount} mal erster Grunt des Tages",
+        "quest_33": "Unbekannt",
```
Comment (123FLO321) [inline resources/webroot/static/data/de.json]: Whats quest 33 actually?
(And mind adding quest_33 and quest_34 in en.json also?

Comment (whitewillem) [reply resources/webroot/static/data/de.json]: 33 = QUEST_BUDDY_FEED
So something like `Feed your buddy %{amount} times`
35, 36, 37 and 38 also excist tho

**Pull request merged**